Program.cs

```diff
@@ -1,57 +1,396 @@
 ﻿using System;
+using System.Linq;
 
 namespace Project1_Group_17
 {
+    /// <summary>
+    /// Project1_Group_17.Program
+    /// Authors: Hunter Bennett, Connor Black, James Dunton
+    /// Date: February 21, 2021
+    /// </summary>
     class Program
     {
+
+        /// <summary>
+        /// The main.
+        /// </summary>
+        /// <param name="args">The args.</param>
         static void Main(string[] args)
         {
             Console.WriteLine("Canadian Cities Data Analayzer © Hunter Bennett, Connor Black, James Dunton");
-            char selection;
+
+            Statistics citiesStats = StartMenu();
+            bool valid = false;
+            citiesStats.DisplayProvinceList();
+            do
+            {
+                string selection;
+                string titleText = "Mode Selection Menu";
+                string dash = new string('-', titleText.Length);
+                Console.WriteLine($"{dash}\n{titleText}\n{dash}\n");
+                Console.WriteLine("\nAvailable selections:");
+                Console.WriteLine("\t1) Display city statistics menu.");
+                Console.WriteLine("\t2) Display province statistics menu.");
+                Console.Write("Please make a selection: ");
+                selection = Console.ReadLine();
+                switch (selection.ToLower())
+                {
+                    case "1":
+                        CityMenu(citiesStats);
+                        break;
+                    case "2":
+                        ProvinceMenu(citiesStats);
+                        break;
+                    case "exit":
+                        System.Environment.Exit(0);
+                        break;
+                    case "restart":
+                        citiesStats = StartMenu();
+                        break;
+                    default:
+                        Console.Write("\nInvalid selection, please key in a valid selection: ");
+                        break;
+                }
+            } while (!valid);
+
+            citiesStats.DisplayLargestPopulationCity("Ontario");
+            citiesStats.DisplaySmallestPopulationCity("Ontario");
+            citiesStats.DisplayProvincePopulation("Ontario");
+            citiesStats.DisplayCityInfo("London");
+            citiesStats.DisplayCityInfo("Windsor");
+            citiesStats.DisplayProvinceCities("Ontario");
```

Comment:
Shouldn't this be removed before merging? This is testing functionality and probably isnt needed after you exit the program